README.md

```diff
@@ -22,6 +22,11 @@ At startup, the agent will communicate with the Docker node it is deployed on vi
 This implementation is using *serf* to form a cluster over a network, each agent requires an address where it will advertise its
 ability to be part of a cluster and a join address where it will be able to reach other agents.
 
+The agent retrieves the IP address it can use to create
+a cluster by inspecting the Docker networks associated to the agent container. If multiple networks are available, it will pickup the first network available and retrieve the IP address inside this network.
+
+Note: Be careful when deploying the agent to not deploy it inside the Swarm ingress network (by not using `mode=host` when exposing ports). This could lead the agent to not be able to create a cluster correctly if picking the IP address inside the ingress network.
```

Comment:
```suggestion
Note: Be careful when deploying the agent to not deploy it inside the Swarm ingress network (by not using `mode=host` when exposing ports). This could lead to the agent being unable to create a cluster correctly, if picking the IP address inside the ingress network.
```